sqlserver/README.md

```diff
@@ -80,13 +92,30 @@ _Available for Agent versions >6.0_
 
 See [Datadog's documentation][9] for additional information on how to configure the Agent for log collection in Kubernetes environments.
 
-#### Linux
 
-Extra configuration steps are required to get the SQL Server integration running on a Linux host:
+#### Containerized
 
-1. Install an ODBC SQL Server Driver, for example the [Microsoft ODBC Driver][10].
-2. Copy the `odbc.ini` and `odbcinst.ini` files into the `/opt/datadog-agent/embedded/etc` folder.
-3. Configure the `conf.yaml` file to use the `odbc` connector and specify the proper driver as specified in the `odbcinst.ini file`.
+For containerized environments, see the [Autodiscovery Integration Templates][22] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                                                                                                                            |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `<INTEGRATION_NAME>` | `sqlserver`                                                                                                                      |
+| `<INIT_CONFIG>`      | blank or `{}`                                                                                                                    |
+| `<INSTANCE_CONFIG>`  | `{"host": "%%host%%,%%port%%", "username": "datadog", "password": "<UNIQUEPASSWORD>", "connector": "odbc", "driver": "FreeTDS"}` |
+
+See the [Autodiscovery template variables documentation][23] to learn how to pass `<UNIQUEPASSWORD>` as an Environment variable instead of a label.
```

Comment:
```suggestion
See [Autodiscovery template variables][23] for details on passing `<UNIQUEPASSWORD>` as an environment variable instead of a label.
```